### ISSUE TITLE
The renderers installed with Hammer should be separate from user-defined renderers.

### ISSUE THREAD

**Issue by ghost** [wikimatze/hammer.vim#40]
Right now, renderers are found through `Hammer::ENV.renderers_path`, which points at `$INSTALL_DIR/renderers`. 
This is bad, though, because when a user wants to add a custom renderer to a location of their choice(outside `$INSTALL_DIR`) they will lose renderers that hammer adds to render markdown, HTML, and POD documents (all of which reside in `$INSTALL_DIR/renderers`). I'm open to suggestions for a solution.

**Comment by ghost** [comment]
This is true for commands also.

**Comment by wikimatze** [comment]
An idea would be to setup an options which enables the user to take a custom folder like `$MYVIMRC/hammer_custom_render` which will be taken instead of the default renderers by the plugin?

**Comment by ghost** [comment]
Yes, something like that could work, I think we need to separate _system renderers (those owned by hammer)_ and _user renderers(those owned by the user)_.

**Comment by wikimatze** [comment]
Got it.
